src/components/dev-hub/controlled-tooltip.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
What's the unused var in this file?

Reply:
There were a few props in the component not being used. I saw a lengthy comment about how this component was used and decided it was safer just to add the comment. We can remove them I have no issue with that.

Reply:
I think they may have a different lint setup where you can't pull out spread items without the linter warning - I'm fine either way

Reply:
I just went ahead and pulled them out